pkg/api/entries.go

```diff
@@ -301,6 +301,16 @@ func createLogEntry(params entries.CreateLogEntryParams) (models.LogEntry, middl
 	logEntry := models.LogEntry{
 		entryID: logEntryAnon,
 	}
+
+	if api.newEntryPublisher != nil {
+		// Publishing notifications should not block the API response.
+		go func() {
+			if err := api.newEntryPublisher.Publish(ctx, logEntryAnon); err != nil {
```

Comment:
one other thought - should we be doing this on the same context as the HTTP call or on a clone of it? If the client closes the connection before the response from `PublishResult.Get()` is successfully read, could we have a situation where the the context was cancelled but we still want to publish the message?

Reply:
I think you're right, good call out!